app/about_us/OurCrew/ourCrew.module.css

```diff
@@ -0,0 +1,63 @@
+.app {
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+    font-size: 22px;
+    text-align: justify;
+    padding: 20px;
+    margin-bottom: 20px;
+    width: 68vw;
+    background-color: #0e1424;
+    box-shadow: 
+      0 4px 8px rgba(0, 0, 0, 0.2), 
+      0 6px 20px rgba(0, 0, 0, 0.2); 
+    background: linear-gradient(145deg, #0e1424,#1c202b); 
+  }
+
+  .description {
+    text-align: center; 
+    margin-bottom: 20px; 
+  }
+
+  .crew{
+    max-width: 70%;
+    height: auto; 
+    border-radius: 8px;
+  }
+
+  .crewName{
+    text-align: center;
+    text-transform: capitalize;
+    font-size: larger;
+  }
+
+  .crewImage {
+    max-width: 80%;
+    height: auto; 
+    border-radius: 8px; 
+  }
+
+  .container {
+    display: flex;
+    align-items: center; 
+    padding: 20px;
+    gap:30px;
+  }
+
+  .left {
```

Comment:
You can achive this alternating effect with the following steps:
1. Remove the `.left` and `.right` classes, and have only one unified class (you can use the `.container` class)
2. Use this css:

```
.container {
  // ... the original style
  flex-directions: row;
}

.container:nth-child(even) {
  // for the even rows override the previous css
  flex-directions: row-reverse;
}
```